contracts/stellar-interchain-token-service/src/migrate.rs

```diff
@@ -93,10 +92,10 @@ pub fn migrate_token(
     };
 
     if let Some(flow_out) = legacy_storage::try_flow_out(env, flow_key.clone()) {
-        storage::set_flow_out(env, token_id.clone(), current_epoch, &flow_out);
+        storage::set_flow_out(env, token_id.clone(), current_epoch, &(flow_out as u128));
```

Comment:
```suggestion
        storage::set_flow_out(env, token_id.clone(), current_epoch, &u128::try_from(flow_out).expect("expected positive"));
```
avoid implicit casts